pkg/controller/common/hash/hash.go

```diff
@@ -36,21 +37,27 @@ func GetTemplateHashLabel(labels map[string]string) string {
 	return labels[TemplateHashLabelName]
 }
 
-// HashObject writes the specified object to a hash using the spew library
-// which follows pointers and prints actual values of the nested objects
-// ensuring the hash does not change when a pointer changes.
-// The returned hash can be used for object comparisons.
-//
+// HashObject returns a hash of a given object using the 32-bit FNV-1 hash function
+// and the spew library to print the object (see WriteHashObject).
 // This is inspired by controller revisions in StatefulSets:
 // https://github.com/kubernetes/kubernetes/blob/8de1569ddae62e8fab559fe6bd210a5d6100a277/pkg/controller/history/controller_history.go#L89-L101
 func HashObject(object interface{}) string {
-	hf := fnv.New32()
+	objHash := fnv.New32()
```

Comment:
any particular reason not to use FNV-1a which has slightly better characteristics

Reply:
No, I've updated to use FNV-1a.